Package.swift

```diff
@@ -1,30 +1,28 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.9
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "hummingbird-compression",
-    platforms: [.iOS(.v12), .tvOS(.v12)],
+    platforms: [.macOS(.v14), .iOS(.v17), .tvOS(.v17)],
     products: [
         .library(name: "HummingbirdCompression", targets: ["HummingbirdCompression"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/hummingbird-project/hummingbird.git", from: "1.0.0"),
-        .package(url: "https://github.com/adam-fowler/compress-nio.git", from: "1.0.0"),
+        .package(url: "https://github.com/hummingbird-project/hummingbird.git", branch: "main"),
```

Comment:
`from: "2.0.0-beta.3"`?

Reply:
Requires changes that are only available in main at the moment